test/fixtures/name_descriptions.yml

```diff
@@ -65,6 +65,7 @@ suillus_desc:
   <<: *DEFAULTS
   name: suillus
   user: dick
+  source_type: <%= NameDescription.source_types[:public] %>
```

Comment:
Repeat the defaults to emphasize that tests depend on these values in these fixtures.